lib/compat/wordpress-5.9/theme.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Additions to the WordPress theme.php file
+ *
+ * @package gutenberg
+ */
+
+if ( function_exists( 'wp_is_block_theme' ) ) {
```

Comment:
I decided to keep the same name as Core to clarify that there's no difference between Gutenberg and Core if available.

Reply:
I assume this was a typo :) Function should be added if it does NOT already exist.
```suggestion
if ( ! function_exists( 'wp_is_block_theme' ) ) {
```